dist/Data-Dumper/Dumper.pm

```diff
@@ -23,9 +26,10 @@ use constant IS_PRE_516_PERL => $] < 5.016;
 use Carp ();
 
 BEGIN {
-    @ISA = qw(Exporter);
-    @EXPORT = qw(Dumper);
-    @EXPORT_OK = qw(DumperX);
+    our @ISA = qw(Exporter);
+    our @EXPORT = qw(Dumper);
+    our @EXPORT_OK = qw(DumperX);
+    our $Useperl;
 
```

Comment:
same for these variables ^

Reply:
The only variable here used outside this scope is `$Useperl`, but since it is aliased again with `our`, it will refer to the same package variable already.

Reply:
This `our` declaration can actually be done inline in the assignment below, thus not needing this extra line.